aggsender/flows/flow_base.go

```diff
@@ -103,8 +89,12 @@ func NewBaseFlow(
 		l1InfoTreeDataQuerier: l1InfoTreeDataQuerier,
 		lerQuerier:            lerQuerier,
 		cfg:                   cfg,
+		timeNowFunc:           TimeNowUTC,
 	}
 }
+func (f *baseFlow) SetTimeNowFunc(timeNow func() uint32) {
```

Comment:
Since this is only used in the `flows` package, can we make this private or remove it, and just set the `baseFlow.timeNowFunc = testTimeNowFn` directly. I do not think it is good to expose this only for tests.